src/js/navigation.js

```diff
@@ -2,12 +2,54 @@ const navigationToggle = document.querySelector(".menu-btn");
 const navigation = document.querySelector("nav");
 const welcome = document.querySelector(".welcome");
 
-navigationToggle.addEventListener("click", ()  => {
+navigationToggle.addEventListener("click", () => {
   document.body.classList.toggle("nav-is-open");
+  welcome.classList.toggle("welcome-to-right");
   welcome.style.transition = "transform ease-out 250ms";
```

Comment:
`welcome.style.transition = "transform ease-out 250ms";` Whay would you not define this in css?

Reply:
sem imel najprej v css na containerju, pa je bil bug v chrome responsive pri kliku na katerikoli nav li

Reply:
Hoces rect, da ma chrome bug?